CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Upload linked PDBs to fix non-IL-stripped symbolication for iOS ([#4527](https://github.com/getsentry/sentry-dotnet/pull/4527))
 - In MAUI Android apps, generate and inject UUID to APK and upload ProGuard mapping to Sentry with the UUID ([#4532](https://github.com/getsentry/sentry-dotnet/pull/4532))
 - Fixed WASM0001 warning when building Blazor WebAssembly projects ([#4519](https://github.com/getsentry/sentry-dotnet/pull/4519))
+- Pin GitHub Actions to Commit-SHA ([#4562](https://github.com/getsentry/sentry-dotnet/pull/4562))
```

Comment:
issue: remove CHANGELOG entry

We add CHANGELOG entries only for user-facing changes, such as new features, improvements and bug fixes, that we ship via our NuGet packages.
Changes that do not affect the SDKs that we produce, such as `"Repository Maintenance"`, `"Flaky Test"` and other dev-only changes, do not warrant a CHANGELOG entry.

For these changes, we add the `#skip-changelog` to the end of the PR description.
See also [CONTRIBUTING.md](https://github.com/getsentry/sentry-dotnet/blob/main/CONTRIBUTING.md#changelog)

TL;DR;
We use the CHANGELOG from a user's perspective, not the perspective of a maintainer/contributor.